jetpack.php

```diff
@@ -93,6 +94,45 @@ function jetpack_admin_unsupported_wp_notice() { ?>
 	return;
 }
 
+/**
+ * Outputs an admin notice for folks running an outdated version of PHP.
+ * @todo: Remove once WP 5.2 is the minimum version.
+ *
+ * @since 7.4.0
+ */
+function jetpack_admin_unsupported_php_notice() { ?>
```

Comment:
I think we could moved this inside the `if ( version_compare( phpversion(), JETPACK__MINIMUM_PHP_VERSION, '<' ) ) {`?